docs/blueprints/EP036.rst

```diff
@@ -130,16 +173,35 @@ This event should sent before any NApp sends its first flow to be published.
     "coloring": {"base": 0}
   }
 
-Deploying pipeline
+
+
+Endpoint API
+============
+  
+  - GET `/api/kytos/multi_table/v2/` - Get all pipelines in the database. Parameters enabled and disabled are allowed.
+
+  - POST `/api/kytos/multi_table/v2/<pipeline_id>/enable` - Enable pipeline to be applied to all switches
+
+  - POST `/api/kytos/multi_table/v2/<pipeline_id>/disable` - Disable pipeline in case of the user wants to return to the default set up.
+
+  - POST `/api/kytos/multi_table/v2/` - Add pipeline to database. Its body is similar to "pipeline.multi_table".
+
+  - DELETE `/api/kytos/multi_table/v2/<pipeline_id>` - Delete a pipeline from database. It should be disabled.

```

Comment:
When it comes to deleting a stored pipeline are we going to allow deletions of only disabled pipelines or how are you thinking about this? I'm thinking for instance about the case where the user is trying to delete a pipeline that's enabled. Maybe always enforcing that they should be disabled first then it'd simplify code/logic.

Reply:
Yes, the pipeline should be always disable before trying to delete it. Going to make it clearer.

Reply:
Cool. Feel free to close this thread once you update the text. Thanks, Aldo.